src/app/shell/shell-routing.module.ts

```diff
@@ -99,6 +104,10 @@ const routes: Routes = [
     canLoad: [ProviderGuard],
     canDeactivate: [CreateGuard]
   },
+  {
+    path: 'create-achievement', component: CreateAchievementComponent,
+    loadChildren: () => import('./personal-cabinet/provider/provider.module').then(m => m.ProviderModule),
```

Comment:
I see you have duplicated this routing? remove one of them
in addition, this component should not be in presonal-cabinet routing. Check please other create-components. there are placed in shell. this is needed to avoid personal cabinet banner under create-form